util/internal/common/src/main/java/org/hibernate/search/v6poc/util/impl/common/CollectionHelper.java

```diff
@@ -47,6 +47,13 @@ private CollectionHelper() {
 		return set;
 	}
 
+	@SafeVarargs
+	public static <T> Set<T> asLinkedHashSet(T... ts) {
+		Set<T> set = new LinkedHashSet<>( ts.length );
```

Comment:
Please use `getInitialCapacityFromExpectedSize()`. `asSet()` is incorrect and should be fixed too.

Reply:
Done